<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDialog</class>
 <widget class="QDialog" name="PasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>328</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qt Bitcoin Trader</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="topMargin">
    <number>9</number>
   </property>
   <item row="0" column="5">
    <widget class="QGroupBox" name="descriptionGroupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox::indicator::checked::hover {background: black; border: 1px solid gray; border-radius: 3px; image: url(:/Resources/Close.png);} QGroupBox::indicator::checked::!hover { background: white; border: 1px solid gray; border-radius: 3px; image: url(:/Resources/Close.png);} QGroupBox:title {background: transparent;}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="accessibleName">
         <string>TR00112</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Your API Key and Secret are crypted by AES256.
Please enter password to decrypt data.
Or reset data and re-enter API keys.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring>passwordEdit</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QGroupBox" name="passInformText">
     <property name="accessibleName">
      <string>TR00106</string>
     </property>
     <property name="title">
      <string>Data Password Protection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="labelPassword">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="accessibleName">
           <string>TR00109</string>
          </property>
          <property name="text">
           <string>Password:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>passwordEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="passwordEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="profileComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonAddProfile">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="removeButton">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelPassword_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="accessibleName">
           <string>TR00108</string>
          </property>
          <property name="text">
           <string>Profile:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>profileComboBox</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="accessibleName">
           <string>EXIT</string>
          </property>
          <property name="text">
           <string>Exit</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="okButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="accessibleName">
           <string>OK</string>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="LogoGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>183</width>
         <height>44</height>
        </size>
       </property>
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="accessibleName">
        <string>LOGOBUTTON</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px;margin:0</string>
       </property>
       <property name="title">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_info">
       <property name="text">
        <string>Version: 1.07.98</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>passwordEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>profileComboBox</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>buttonAddProfile</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>descriptionGroupBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>passwordEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwordEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>okButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>PasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>PasswordDialog</receiver>
   <slot>resetDataSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAddProfile</sender>
   <signal>clicked()</signal>
   <receiver>PasswordDialog</receiver>
   <slot>addNewProfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>PasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwordEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PasswordDialog</receiver>
   <slot>checkToEnableButton(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetDataSlot()</slot>
  <slot>checkToEnableButton(QString)</slot>
  <slot>addNewProfile()</slot>
  <slot>okPressed()</slot>
 </slots>
</ui>
